Sources/Core/Console.swift

```diff
@@ -63,6 +63,15 @@ extension TerminalController: TerminalControlling {
     }
 }
 
+class PrintController: TerminalControlling {
+    func endLine() {}
```

Comment:
Would it make sense here to "print" a line break ? 🤔

Reply:
Yes, good catch. Looking at the `TerminalController` `endLine()` method it adds a `\n`. Adding a print here will mirror the intended behavior.